README.md

```diff
@@ -27,3 +27,21 @@ so I can easily explore the options for my treatment.
 As a young person suffering from depression,  
 I would like to be able to keep track of the options I am interested in,  
 so I can discuss them with my clinician afterwards.  
+
+### Technical details
+
+Handlebars.registerHelper('test', function(treatments, options) {
+  return treatments.medication.name;
+});
+
+<!--<h2>{{test treatments}}</h2>-->
```

Comment:
ooops! It's a mistake, yesterday I had some difficulties with templating, so I used a simple test helper, and I forgot to remove it. I can't remember why I put that into the readme :(...